config/settings/local.py

```diff
@@ -21,6 +21,10 @@
 
 INTERNAL_IPS = ['127.0.0.1']
 
+# create directory for log file
+filepath = env('LOG_FILE_PATH')
```

Comment:
we only need to do this for production..

Reply:
done